CIPs/0010.md

```diff
@@ -1,43 +1,91 @@
 # CIP 0010: General Account Signer Authorization
 
 - Date: 2020-05-13
-- Author: @nambrot
+- Author: @nambrot @kphed @eko-mirhard
```

Comment:
hi @nambrot @kphed  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.